infra/charts/turing/values.yaml

```diff
@@ -125,7 +125,8 @@ turing:
         BuildNamespace: default
         BuildTimeoutDuration: 20m
         DestinationRegistry: ghcr.io
-        BaseImageRef: ghcr.io/gojek/turing/pyfunc-ensembler-service:latest
+        BaseImageRef:
+          3.7.*: ghcr.io/gojek/turing/pyfunc-ensembler-service:latest
```

Comment:
Updating the base values to match the new format. Will capture the mapping for the new python versions and the corresponding public artifacts in chart version `0.2.9`.